workload-team/cluster-stamp.bicep

```diff
@@ -28,6 +28,7 @@ param clusterAuthorizedIPRanges array = []
 @description('AKS Service, Node Pool, and supporting services (KeyVault, App Gateway, etc) region. This needs to be the same region as the vnet provided in these parameters. This defaults to the resource group\'s location for higher reliability.')
 param location string = resourceGroup().location
 
+@description('The Kubernetes version for the AKS cluster.')
 param kubernetesVersion string = '1.30'
```

Comment:
I know this is not part of this PR, but since we added a description to this I was wondering whether we should add allowed values as well or simple convert this into a variable.

Feel free to mark as won't fix if this is not too relevant at this point in time.

Reply:
I added a description because it's the only param without a description currently, so it was purely for consistency.

Thinking about it, maybe this should be a var instead of a param - we don't necessarily want people changing the version and expecting it to work. By using a var, we indicate that this is the version we've tested with. What do you think?

Reply:
I would say yes to a var instead

Reply:
Updated.